client/src/components/DataFiles/DataFilesSearchbar/DataFilesSearchbar.js

```diff
@@ -97,10 +100,12 @@ DataFilesSearchbar.propTypes = {
   scheme: PropTypes.string.isRequired,
   system: PropTypes.string.isRequired,
   /** Additional `className` (or transpiled `styleName`) for the root element */
-  className: PropTypes.string
+  className: PropTypes.string,
+  publicData: PropTypes.bool
```

Comment:
the `onClear` sends you to the workbench which can be a problem for unauthed users.

Reply:
Good catch, for now I'm removing the button in the unauth'd public view.